src/events/autoMod.js

```diff
@@ -0,0 +1,25 @@
+const fs = require("fs");
+
+class CacheManager {
+     constructor(val) {
+          this.cache = String(val).split(/[\n\r]/);
+          this.cache = this.cache.filter((url) => url.length > 0);
+     }
+     hasNot(val) {
+          for (const url of this.cache)
+               if (new RegExp(url).test(val)) return false;
```

Comment:
wow this looks like its going to be slow (relatively speaking obv)

Wouldn't it be much nicer to store all of the URLs in a Map? Map.has is very performant

Reply:
This 
+ please use brackets, gives me anxienty
+ since the RegExp().test() returns a bool, it does not need to be wrapped in a if condition, just return the RegExp - but its being replaced anyways

Reply:
I will fix all of that tomorrow (or today at night), thanks guys for telling!

Reply:
Resolved.